Runtime/DataSchemas/RomuloInternalSchema.cs

```diff
@@ -1,111 +1,104 @@
 using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
 using ReupVirtualTwin.helpers;
+using System;
 
 namespace ReupVirtualTwin.dataSchemas
 {
     public static class RomuloInternalSchema
     {
-        public static JObject materialChangeInfo { get; private set; }
-        public static JObject sceneStateSchema { get; private set; }
-        public static JObject sceneStateAppearanceSchema { get; private set; }
-        public static JObject objectWithChangedMaterialSceneSchema { get; private set; }
-        public static JObject objectWithChangedColorSceneSchema { get; private set; }
-        public static JObject objectWithNoChangesSceneSchema { get; private set; }
-        public static JObject materialSchema { get; private set; }
+        public static JSchema boolSchema { get; private set; }
+        public static JSchema intSchema { get; private set; }
+        public static JSchema stringSchema { get; private set; }
+        public static JSchema nullSchema { get; private set; }
+        public static JSchema materialChangeInfoSchema { get; private set; }
+        public static JSchema sceneStateSchema { get; private set; }
+        public static JSchema sceneStateAppearanceSchema { get; private set; }
+        public static JSchema objectWithChangedMaterialSceneSchema { get; private set; }
+        public static JSchema objectWithChangedColorSceneSchema { get; private set; }
+        public static JSchema objectWithNoChangesSceneSchema { get; private set; }
+        public static JSchema materialSchema { get; private set; }
 
         static RomuloInternalSchema()
         {
+            stringSchema = JSchema.Parse(@"{ ""type"": ""string"" }");
 
-            materialSchema = new()
-            {
-                { "type", DataValidator.objectType },
-                { "properties", new JObject
-                    {
-                        { "id", DataValidator.intSchema },
-                        { "texture", DataValidator.stringSchema },
-                        { "widthMilimeters", DataValidator.intSchema },
-                        { "heightMilimeters", DataValidator.intSchema },
-                    }
+            intSchema = JSchema.Parse(@"{ ""type"": ""integer"" }");
+
+            boolSchema = JSchema.Parse(@"{ ""type"": ""boolean"" }");
+
+            nullSchema = JSchema.Parse(@"{ ""type"": ""null"" }");
+
+            materialSchema = JSchema.Parse(@"{
+                ""type"": ""object"",
+                ""properties"": {
+                    ""id"": { ""type"": ""string"" },
```

Comment:
isn't this supposed to be an integer? is there a reason it was changed?
```suggestion
                    ""id"": { ""type"": ""integer"" },
```